src/clj/chronograph/db/organization.clj

```diff
@@ -16,11 +16,22 @@
                    :updated-at now}
                   db/sql-opts))))
 
+(defn where
+  ([options]
+   (where db/datasource options))
+  ([tx options]
+   (sql/find-by-keys tx :organizations options db/sql-opts)))
+
+(defn find [tx options]
```

Comment:
Same observation as this comment: https://github.com/nilenso/chronograph/pull/11/files#r498171364